ContactDetailsApi/V2/Factories/EntityFactory.cs

```diff
@@ -14,10 +14,13 @@ public static class EntityFactory
     {
         public static ContactDetails ToDomain(this ContactDetailsEntity databaseEntity)
         {
+            if (databaseEntity == null) return null;
+            if (databaseEntity.ContactInformation == null) return null;
             var contactInformation = databaseEntity.ContactInformation;
             if (contactInformation.ContactType == V1.Domain.ContactType.address &&
-                string.IsNullOrEmpty(contactInformation.AddressExtended.AddressLine1))
+                string.IsNullOrEmpty(contactInformation?.AddressExtended?.AddressLine1))
             {
+                if (contactInformation?.AddressExtended == null) return null;
```

Comment:
Same question as for the `ContactInformation` about: whether the overall `ContactDetailsEntity` should be null when one of its nested models is `null`.

If intended, it's probably worth mentioning in a PR description, or even better, a code comment - documenting the business logic behind it.